app/assets/javascripts/discourse/app/widgets/link.js

```diff
@@ -67,7 +67,18 @@ export default createWidget("link", {
     if (attrs.icon) {
       if (attrs["aria-label"]) {
         let icon = iconNode(attrs.icon);
-        icon.properties.attributes["aria-label"] = I18n.t(attrs["aria-label"]);
+
+        if (attrs.ariaLabelOptions) {
+          icon.properties.attributes["aria-label"] = I18n.t(
+            attrs["aria-label"],
+            attrs.ariaLabelOptions
+          );
+        } else {
+          icon.properties.attributes["aria-label"] = I18n.t(
+            attrs["aria-label"]
+          );
+        }
```

Comment:
Can this be expressed like this: (or are there non-undefined falsy `attrs.ariaLabelOptions` values that cause problems?)

```suggestion
        icon.properties.attributes["aria-label"] = I18n.t(
          attrs["aria-label"],
          attrs.ariaLabelOptions
        );
```